src/pytorch_lightning/core/module.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 """The LightningModule - an nn.Module with many additional features."""
 
-import collections
+import collections.abc
```

Comment:
importing just `collections` should be enough, no?

Reply:
no, as given [here](https://docs.python.org/3.9/library/collections.html#module-collections):
> Deprecated since version 3.3, will be removed in version 3.10: Moved [Collections Abstract Base Classes](https://docs.python.org/3.9/library/collections.abc.html#collections-abstract-base-classes) to the [collections.abc](https://docs.python.org/3.9/library/collections.abc.html#module-collections.abc) module. For backwards compatibility, they continue to be visible in this module through Python 3.9.

Reply:
I did this to avoid the huge squiggly lines in the IDE :)